crispr_read_counts/single_guide_merge.py

```diff
@@ -65,3 +88,21 @@ def get_counts_results(in_f: TextIO, with_or_without_plasmid_counts: bool):
       get_counts_results(in_f, has_plasmid)
 
   return sample_name, plasmid_name, sample, plasmid, targeted_genes
+
+
+def write_stats_to_file(sample_count: Dict[str, int], output_path: str):
+  stats = {
+    'zero_count_guides': 0,
+    'low_count_guides': 0,
+    'total_counts': 0
+  }
+
+  for a_count in sample_count.values():
+    stats['total_counts'] += a_count
+    if a_count == 0:
+      stats['zero_count_guides'] += 1
+    if a_count < SingleGuideReadCounts.LOW_COUNT_GUIDES_THRESHOLD:
```

Comment:
An `elif` here would save some nanoseconds.

Reply:
low_count_guide should include zero_count_guide, `elif` will exclude them.

Reply:
I see, okay then.